packages/puppeteer/package.json

```
@@ -34,7 +34,7 @@
     "express": "^4.16.4",
     "mocha": "^5.2.0",
     "nyc": "^14.1.1",
-    "puppeteer": "^5.3.1",
+    "puppeteer": "9.1.0",
```

Do we want this pinned at `9.1.0`?

```suggestion
    "puppeteer": "^9",
```